kubernetes/assets/recommended_monitors/monitor_containers_terminated.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "[kubernetes] Monitor Kubernetes Pods Terminated",
+	"type": "query alert",
+	"query": "change(sum(last_5m),last_5m):avg:kubernetes.containers.state.terminated{*} by {kubernetes_cluster} >= 1",
+	"message": "Containers in pods terminated on {{kubernetes_cluster.name}}",
+	"tags": [
+		"integration:kubernetes"
+	],
+	"options": {
+		"notify_audit": true,
+		"locked": false,
+		"timeout_h": 0,
+		"include_tags": true,
+		"no_data_timeframe": 5,
+		"require_full_window": true,
+		"new_host_delay": 300,
+		"notify_no_data": true,
+		"renotify_interval": 0,
+		"escalation_message": "",
+		"thresholds": {
+			"critical": 1,
+			"critical_recovery": 0
+		}
+	},
+	"recommended_monitor_metadata": {
+		"description": "Notify your team when the number of terminated containers for Kubernetes is more than 1."
```

Comment:
Aren't terminated containers a normal part of using Kubernetes? Why alerting on that?